src/CTA.WebForms2Blazor/Helpers/FilePathHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace CTA.WebForms2Blazor.Helpers
+{
+    public static class FilePathHelper
+    {
+        private const string FileNameDoesNotContainExtensionError = "Cannot alter file name, file name {0} does not contain extension {1}";
+
+        public static string AlterFileName(string oldFilePath, string newFileName = null, string oldExtension = null, string newExtension = null)
+        {
+            var actualOldExtension = oldExtension ?? Path.GetExtension(oldFilePath);
+            var oldFileDirectory = Path.GetDirectoryName(oldFilePath);
+            var oldFileName = Path.GetFileName(oldFilePath);
+
+            if (!oldFileName.Contains(actualOldExtension))
+            {
+                throw new ArgumentException(string.Format(FileNameDoesNotContainExtensionError, oldFileName, actualOldExtension));
+            }
```

Comment:
I suggest using `.EndsWith()` instead of `.Contains()` because there could be an edge case where `oldExtension` exists elsewhere in `oldFilePath`.